spec/scripts/changelog_check_spec.rb

```diff
@@ -0,0 +1,63 @@
+require 'rails_helper'
+require_relative '../../scripts/changelog_check'
+
+GIT_FIXTURES = YAML.safe_load(File.read(File.expand_path('spec/fixtures/git_log_changelog.yml')))
```

Comment:
❤️ spec!!! thank you!!!

This defines a constant in the global namespace. I'd add an overall describe and make it a local in there (or even a `let`/helper method, since reading the file a few times is a fairly cheap operation


```suggestion
RSpec.describe 'scripts/changelog_check' do
  git_fixtures = YAML.safe_load(File.read(File.expand_path('spec/fixtures/git_log_changelog.yml')))
```